frontend/package.json

```diff
@@ -52,6 +52,7 @@
     "eslint-config-next": "^12.2.5",
     "eslint-plugin-jest-dom": "^4.0.2",
     "eslint-plugin-testing-library": "^5.6.0",
+    "fast-check": "^3.1.2",
```

Comment:
This is a new dependency. I'll comment elsewhere what I'm using it for, but some reasons why I think it's a safe addition:

- It's only a `devDependency` that is only run during tests (which can still inspect and modify env vars, but shouldn't otherwise impact our shipped code).
- It only has a single transitive dependency.
- It has existed for quite a while already and has been quite stable in that time (I've also used this [at my previous employer](https://github.com/inrupt/solid-client-js/blob/24043d1bf3a09fe88c98a2224f4bbbf9075500b3/package.json#L177) without major issues.
- I've followed [the author on Twitter](https://twitter.com/ndubien/) for a while, who appears to be a real and trustworthy person :)